spec/lib/jobs_spec.rb

```diff
@@ -2,7 +2,12 @@
 
 describe Jobs do
   describe '.search(options)' do
-    subject(:search) { Jobs.search(Keyword: 'jobs') }
+    subject(:search) do
+      Jobs.search({ query:'jobs',
+                    organization_code: '',
+                    location_name: 'Washington, DC, United States',
+                    results_per_page: 10})
+    end
```

Comment:
@peggles2 , this would be a good time to add a spec to check that we're searching with the right params (which you can build on for the Radius story we discussed). Something like:
```ruby
it 'searches USAJOBS with the correct params' do
  search
  expect(a_request(:get, usajobs_url).with( Keyword: 'jobs'...etc. )).to have_been_made
end
```
Check out the documentation on using the Webmock gem for this: 
https://github.com/bblimke/webmock#setting-expectations-in-rspec-with-a_request

You might have to play with the expectation a bit; let me know if you want to pair.

Reply:
We can add that to jobscrub story.  It be out of scope for this project since its not adding any additional params. Would you like to place that in the Jiraticket?

Reply:
It's in scope for this story, because you're changing how and where we're building the params that we're passing to USAJOBs. For example, if you're calling `Jobs.search(query: 'foo')`, we need to verify that we are passing `Keyword: 'foo'` to USAJOBs.

Reply:
This will take some time to do. Rather not hold jobscrub over this. Lets code review the rest and we can look at this one together. Also this get request requires to pass authentication etc.

Reply:
After our discussion added the test. I made a change to it expecting 10 results. There are currently 3 nursing jobs at NIH. Also can't put a abcd for organization because usajobs will return nothing. I changed it to HE38 which is NIH.